<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefSubtitles</class>
 <widget class="QWidget" name="PrefSubtitles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QGroupBox" name="groupBox_autoload">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>448</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Autoload</string>
   </property>
   <layout class="QGridLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="1" column="1">
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>21</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="autoload_label">
      <property name="text">
       <string>Autoload subtitles files (*.srt, *.sub...):</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="buddy">
       <cstring>font_autoload_combo</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="font_autoload_combo">
      <item>
       <property name="text">
        <string>Same name as movie</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>All subs containing movie name</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>All subs in directory</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_encoding">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>448</width>
     <height>104</height>
    </rect>
   </property>
   <property name="title">
    <string>Encoding</string>
   </property>
   <layout class="QVBoxLayout" name="_2">
    <item>
     <layout class="QHBoxLayout" name="_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="encoding_label">
        <property name="text">
         <string>Default subtitle encoding:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>font_encoding_combo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="font_encoding_combo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="_4">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="use_enca_check">
        <property name="text">
         <string>Try to autodetect for this language:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="enca_lang_combo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
